### DISCUSSION TITLE
Can businesses use the community edition?

### DISCUSSION THREAD

**Discussion by yswf** [dbeaver/dbeaver#13520] [Q&A] [answered]
I have been using DBeaver Community to manage the database when I was in college. Can I continue to use the community version after entering the enterprise? Because the company is not a pure software development company, the software purchase approval process is more troublesome.I am not currently using it. I would like to know if using DBeaver Community in the company will infringe.

**Comment by p91paul** [comment, marked as answer]
Dbeaver Community is free software under the Apache License (https://dbeaver.io/about/). I'm neither a lawyer nor a DBeaver developer, but it's safe to assume you can use DBeaver however you like because that license does not pose restrictions on usage.
Besides, plenty of people report issues on the community edition plainly stating they use it for work, and nobody on the DBeaver team complains. Instead, they fix bugs or implement the requested features.

Of course the Enterprise License allows you to have a nice set of additional features.

**Reply by tati-kru** [reply]
Thank you @p91paul!

@yswf DBeaver Community Edition is distributed with Apache 2.0 license. It means that you can use DBeaver CE in your corporate environment without any limitations. However, we highly recommend to use the commercial DBeaver versions in the companies because of better security and stability of the products.

**Reply by yswf** [reply]
Thank you for your answers, I have always wanted to use it.😁 
I have already submitted an application for purchase requirements to the company, but I don't know if it is passed.😏
